web/viewer.html

```diff
@@ -137,8 +137,13 @@
             <input type="checkbox" id="findMatchCase" class="toolbarField" tabindex="95">
             <label for="findMatchCase" class="toolbarLabel" data-l10n-id="find_match_case_label">Match case</label>
           </div>
+          <div id="findbarOptionsThreeContainer">
```

Comment:
It looks like you flipped the names of the containers here, since two comes before three :-)

More generally, you now have a container with only *one* input element which doesn't really make all that much sense. I'd expect the following format instead:
```javascript
          <div id="findbarOptionsTwoContainer">
            <input type="checkbox" id="findMatchDiacritics" class="toolbarField" tabindex="96">
            <label for="findMatchDiacritics" class="toolbarLabel" data-l10n-id="find_match_diacritics_label">Match Diacritics</label>
            <input type="checkbox" id="findEntireWord" class="toolbarField" tabindex="97">
            <label for="findEntireWord" class="toolbarLabel" data-l10n-id="find_entire_word_label">Whole words</label>
          </div>
          <div id="findbarOptionsThreeContainer">
            <span id="findResultsCount" class="toolbarLabel hidden"></span>
          </div>
```